### DISCUSSION TITLE
"usbipd: error: WSL 'usbip' client not correctly installed." => can't attach

### DISCUSSION THREAD

**Discussion by LuciMain** [dorssel/usbipd-win#733] [Q&A] [answered]
Hi

i always used usbipd wsl attach --busid=1-12 and that worked fine. But It seems like since yesterday it doesn't work anymore and I get the error: "usbipd: error: WSL 'usbip' client not correctly installed. See https://github.com/dorssel/usbipd-win/wiki/WSL-support for the latest instructions."
What can I do to get it to work again?

**Comment by dorssel** [comment, marked as answer]
You've probably updated your repo (Ubuntu? `sudo apt upgrade`?), and a different `usbip` version was installed. Just rerun the `update-alternatives` from the instructions.

**Reply by LuciMain** [reply]
Worked! Thanks!
